src/download.js

```diff
@@ -11,7 +11,10 @@ SkynetClient.prototype.download = function (skylink, customOptions = {}) {
   const opts = { ...defaultDownloadOptions, ...customOptions, download: true };
   const url = this.getDownloadUrl(skylink, opts);
 
-  window.open(url, "_blank");
+  const downloadElement = document.createElement("a");
+  downloadElement.setAttribute("href", url);
+  downloadElement.setAttribute("download", true);
+  downloadElement.click();
```

Comment:
This should do the trick. Make sure that the url has `attachment=true` appended - this will force siad to return the download content disposition that should force the browser to download.

```suggestion
  window.location = url;
```

Reply:
If that still doesn't work even after you confirm that `attachment=true` param is set in the url, check the content disposition on response headers on portal and on siad to verify that those are working correctly. The workaround with creating new element will not be needed to fix this.

Reply:
Yeah forgot about that option. That's way better